README.md

```diff
@@ -84,12 +84,12 @@ Is your Git repository bursting at the seams?
 
     No options are required. You can learn about available options by typing `git sizer -h` or by reading on.
 
-    (The above command assumes that you have added `git-sizer` to your `PATH`. If you don't add it to your `PATH`, you need to type its full path and filename to run it; e.g., `/path/to/bin/git-sizer`.)
+    *Note*: The above command assumes that you have added `git-sizer` to your `$PATH`. If you don't add it to your `PATH`, you need to type its full path and filename to run it; e.g., `/path/to/bin/git-sizer`. From now on, we will use `git sizer` which is interchangeable with `git-sizer` anywhere (assuming you added `git-sizer` to your `$PATH` already, which is recommended).
 
 
 ## Usage
 
-By default, `git-sizer` outputs its results in tabular format. For example, let's use it to analyze [the Linux repository](https://github.com/torvalds/linux), using the `--verbose` option so that all statistics are output:
+By default, `git sizer` outputs its results in tabular format. For example, let's use it to analyze [the Linux repository](https://github.com/torvalds/linux), using the `--verbose` option so that all statistics are output:
```

Comment:
I've been using `git-sizer` as the name of the project and of the program (for example when discussing it in text), even though it can usually be run by typing `git sizer`. I'm trying to be consistent about that. So I don't like this change, or others like it.

Reply:
Whichever you like, it needs be consistent within the whole document.